spec/models/service_ansible_playbook_spec.rb

```diff
@@ -120,6 +120,8 @@
         basic_service.preprocess(action)
         service.reload
         expect(basic_service.options[:provision_job_options][:hosts]).to eq("default_host1,default_host2")
```

Comment:
expect(basic_service.options[:provision_job_options]).to include(
  :hosts            => "default_host1,default_host2",
  :credential       => credential_0.native_ref,
  :vault_credential => credential_3.native_ref
)